prometheuscollector/prometheuscollector.go

```diff
@@ -46,11 +46,11 @@ var (
 )
 
 type Collector struct {
-	metrics tusd.Metrics
+	metrics *tusd.Metrics
```

Comment:
With the code I just pushed (https://github.com/tus/tusd/pull/105/commits/13bc64517ba84741ad26ade44c1e8aa3ed451eba) I get the following output:
```
go vet ./prometheuscollector
prometheuscollector/prometheuscollector.go:54: New passes lock by value: github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
prometheuscollector/prometheuscollector.go:56: literal copies lock value from metrics: github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
prometheuscollector/prometheuscollector.go:60: Describe passes lock by value: prometheuscollector.Collector contains github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
prometheuscollector/prometheuscollector.go:69: Collect passes lock by value: prometheuscollector.Collector contains github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
prometheuscollector/prometheuscollector.go:115: Handler passes lock by value: github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
prometheuscollector/prometheuscollector.go:116: call of New copies lock value: github.com/tus/tusd.Metrics contains github.com/tus/tusd.ErrorsTotalMap
exit status 1
```

But it seems to be fine on [AppVeyor](https://ci.appveyor.com/project/Acconut/tusd/build/1.0.421)